content/ui-patterns/navigation.mdx

```diff
@@ -0,0 +1,392 @@
+
+When being used as navigation, activating a segmented control option may or may not change the URL.
+
+[Segmented control component guidelines](/components/segmented-control)
+
+</Box>
+
+<img
+    width="456"
+    alt="Screenshot of code/blame/preview on .md files"
+    src="https://github.com/primer/design/assets/2313998/0b1e308d-a351-4457-8214-d863d8f76635"
+/>
+
+</Box>
+
+## Layout
+
+Layout is used to structure your content in a way that reflects the hierarchy of the navigation within the app. Navigational elements should be laid out in close proximity to the content they affect.
+
+For example, pagination should go directly below the collection it navigates.
+
+### Layouts for navigating in the same context
+
+#### Parent-detail split page layout
+
+<img
+    width="960"
+    alt="Screenshot of a repo's security advisories page with the sidebar and main areas highlighted"
+    src="https://github.com/primer/design/assets/2313998/7d8bafeb-f971-45d2-ac0e-7850d7f81912"
+/>
+
+A "parent" sidebar with a nav list or tree view that affects "detail" content. The "parent" navigation remains on screen while the user navigates between different "detail" views.
+
+On narrow viewports, the parent list may be stacked above the detail view or turned into an action menu. See the [responsive layout guidelines](/foundations/layout#page-types) for more info.
+
+#### Tabbed interfaces
+
+<img
+    width="960"
+    alt="Screenshot of a repo's security advisories page with the tabs and tab panel areas highlighted"
+    src="https://github.com/primer/design/assets/2313998/f7fb7251-f9bf-4d58-b6ec-16b66f3959da"
+/>
+
+Tabbed interfaces enable seamless navigation between different subpages within a unified context. Tabs go directly above the content they affect.
+
+### Headings and hierarchical page sections
+
+<img
+    width="960"
+    alt="Wireframe of a settings page with multiple nested sections"
+    src="https://github.com/primer/design/assets/2313998/88dedda4-355c-48ba-af14-9c01934dd276"
+/>
+
+A [page header](/components/page-header) is the title area at the top of the main content region that shows the user where they are. It may also to contain navigational elements like breadcrumbs or a link to go up a level of hierarchy.
+
+Use section headings to show the user where they are in the context of the page. These headings should be styled to reflect their hierarchy within the page. Page section headings should be marked up with the appropriate heading level (`h3`–`h6`).
+
+See our [typographic style options](/primitives/storybook/?path=/story/typography-functional--overview) for heading styles.
+
+
+## Accessibility
+
+### Skip links
+
+Facilitate efficient navigation for screen reader users by including skip links to bypass dense redundant content and navigate directly to essential sections.
```

Comment:
Do we have an a11y article on `skip links` to link to?

Reply:
Hmm maybe? @ericwbailey - do you know?

Reply:
Here's a couple:

- [Incorporating Skip to Main Content Links for Better Website Accessibility - The A11Y Collective](https://www.a11y-collective.com/blog/skip-to-main-content/)
- [How-to: Use skip navigation links - The A11Y Project]

Reply:
So it looks like we don't have anything internal. @lukasoppermann - do you think it's worth linking to one of these external articles?

Reply:
I'm also happy to write up some internal documentation, if y'all find it helpful!